### ISSUE TITLE
_set as method name throws error

### ISSUE THREAD

**Issue by gitawego** [6to5/6to5#610]
I don't know if it's intended, if in my object, I have _set as method name, 6to5 throws error:
```js
var helper = {
   _set(key, data){
     }
};
```
this code will be converted to:
```js
_set: function _core.Set(key){

}
```
this gives  ``Unexpected token .``  error.

**Comment by sebmck** [comment]
Looks like there's two bugs here. One is that `types.isReferenced` isn't ignoring function and class ids and `_set` is somehow being transformed to `Set`.

**Comment by sebmck** [comment]
Fixed and released as of 3.0.9, thanks!

**Comment by gitawego** [comment]
you are welcome. updating via npm ^^
